### PR TITLE
feat(contract-standards): add events to FT contract standard

### PR DESCRIPTION
**Pull request by itegulov**
As minting and burning are not a part of FT contract standard yet, these events are not being emitted

### CODE REVIEW

File: near-contract-standards/src/non_fungible_token/events.rs [hunk 1]
```diff
@@ -7,17 +7,17 @@
 //! This is an extension of the events format (nep-297):
 //! <https://github.com/near/NEPs/blob/master/specs/Standards/EventsFormat.md>
 //!
-//! The three events in this standard are [`NftMintData`], [`NftTransferData`], and [`NftBurnData`].
+//! The three events in this standard are [`NftMint`], [`NftTransfer`], and [`NftBurn`].
```
Comment (ChaoticTempest) [inline near-contract-standards/src/non_fungible_token/events.rs:10]: Nice, thanks for the catching this!

**Review by ChaoticTempest — APPROVED** [review]
Looks pretty straightforward to me, but I'll leave it to @austinabell to have the final say since he might point out some serialization stuff I might not know of

**Review by austinabell — APPROVED** [review]
Awesome!

**Comment by austinabell** [comment]
Hey @frol and/or @telezhnaya I should have pinged you guys to make sure this all matches up with what gets picked up by the indexer. Do you guys mind having a quick look before I release this?

**Comment by frol** [comment]
@austinabell Sorry for the delay. This implementation is compatible with the NEP, and I have even checked how it works with the FT example: https://github.com/near-examples/FT/pull/106

File: near-contract-standards/src/fungible_token/events.rs [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        let amount = &U128(100);
+        FtTransfer { old_owner_id, new_owner_id, amount, memo: None }.emit();
+        assert_eq!(
+            test_utils::get_logs()[0],
+            r#"EVENT_JSON:{"standard":"nep141","version":"1.0.0","event":"ft_transfer","data":[{"old_owner_id":"bob","new_owner_id":"alice","amount":"100"}]}"#
+        );
+    }
+
+    #[test]
+    fn nft_transfers() {
```
[the remark this answers is not in the document]
Comment (telezhnaya) [reply near-contract-standards/src/fungible_token/events.rs:197]: `ft_transfers` 🙂

File: near-contract-standards/src/fungible_token/events.rs [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        let amount = &U128(100);
+        FtBurn { owner_id, amount, memo: None }.emit();
+        assert_eq!(
+            test_utils::get_logs()[0],
+            r#"EVENT_JSON:{"standard":"nep141","version":"1.0.0","event":"ft_burn","data":[{"owner_id":"bob","amount":"100"}]}"#
+        );
+    }
+
+    #[test]
+    fn nft_burns() {
```
Comment (telezhnaya) [inline near-contract-standards/src/fungible_token/events.rs:171]: `ft_burns`

**Review by telezhnaya — COMMENTED** [review]
Thank you!

File: near-contract-standards/src/fungible_token/events.rs (hunk 3 above)
Comment (telezhnaya) [reply near-contract-standards/src/fungible_token/events.rs:171]: Oh it's merged 😅 I just noticed that
Created PR with the fix https://github.com/near/near-sdk-rs/pull/735

**Pull request merged**